src/databricks/labs/ucx/runtime.py

```diff
@@ -24,6 +24,104 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(hms_db) -> str:
+    return f"""CREATE
+OR REPLACE VIEW {hms_db}.vw_failure_summary AS WITH failuretab (failure, issue, object_type) AS (
+  SELECT
+    explode(from_json(failures, 'array<string>')) AS failure,
+    substring_index(failure, ":", 1) issue,
```

Comment:
Why do we need this? It's not part of the original issue. We want to show all failures, without post-processing them

Reply:
I think in that case we should have two views - one for summary as it is right now and the second one is more like a drill down with details such as object ids.

Reply:
@nfx

Reply:
agreed,  i made it as a detail view and selecting summary for assessment report.

Reply:
I am still keeping the object type for summary.